test/integration/routes_test.rb

```diff
@@ -11,4 +11,34 @@ class RoutesTest < LegacyIntegrationTest
   should "route to new downtimes controller new downtime" do
     assert_routing("/editions/1/downtime/new", controller: "downtimes", action: "new", edition_id: "1")
   end
+
+  context "allowed content types for new design system" do
+    %i[answer_edition help_page_edition].each do |content_type|
+      context content_type do
+        setup do
+          test_strategy = Flipflop::FeatureSet.current.test!
+          test_strategy.switch!(:design_system_edit, true)
+          @edition = FactoryBot.create!(content_type)
+        end
```

Comment:
I think we should be turning the feature toggle off in a `teardown` to ensure the value doesn't bleed between tests.